mlflow/store/tracking/file_store.py

```diff
@@ -908,12 +912,8 @@ def _list_run_infos(self, experiment_id, view_type):
                     continue
                 if LifecycleStage.matches_view_type(view_type, run_info.lifecycle_stage):
                     run_infos.append(run_info)
-            except MissingConfigException as rnfe:
-                # trap malformed run exception and log warning
-                r_id = os.path.basename(r_dir)
-                logging.warning(
-                    "Malformed run '%s'. Detailed error %s", r_id, str(rnfe), exc_info=True
-                )
+            except MissingConfigException:
+                pass
```

Comment:
can we check if `r_dir` is a reserved name and skip it?

Reply:
We skip reserved named at line 892 already. This warns for artifacts folder, where only uuid/artifacts folder exists

Reply:
do we need this change?